pkg/kubectl/cmd/portforward.go

```diff
@@ -109,6 +116,7 @@ func NewCmdPortForward(f cmdutil.Factory, cmdOut, cmdErr io.Writer) *cobra.Comma
 		},
 	}
 	cmdutil.AddPodRunningTimeoutFlag(cmd, defaultPodPortForwardWaitTimeout)
+	cmdutil.AddAddressFlag(cmd)
```

Comment:
prefer binding to the Address field in the struct, and defaulting to "127.0.0.1", "::1" above when constructing `opts` above

```
cmd.Flags().StringSliceVar(&opts.Addresses, "address", opts.Addresses, "Addresses to listen on (comma separated)")
```